server/problems/problems.js

```diff
@@ -1,9 +1,18 @@
 import {Router} from 'express';
 import { problems } from './mock';
+import bodyParser from 'body-parser';
+
 const router = Router();
 
+router.use(bodyParser.json());
```

Comment:
this middleware should be defined globally for all our API,
now body is parsed only for /api/problems pattern